packages/grafana-ui/src/components/Select/SelectBase.tsx

```diff
@@ -212,7 +215,7 @@ export function SelectBase<T>({
     'aria-label': ariaLabel,
     autoFocus,
     backspaceRemovesValue,
-    captureMenuScroll: false,
+    captureMenuScroll,
```

Comment:
```suggestion
    captureMenuScroll: onMenuScrollToBottom || onMenuScrollToTop,
```
Maybe do this, so we don't have to expose the extra `captureMenuScroll` prop. If you have any reason for not doing this, please do tell! :slightly_smiling_face: